client/reader/site-stream/index.jsx

```diff
@@ -127,10 +128,6 @@ const SiteStream = React.createClass( {
 			title = this.translate( 'Loading Site' );
```

Comment:
I guess it has to do with changing from `setTitle` action calls (either in the controller or in a component's `render()`) to using `<DocumentHead>` (where in turn the action is called the lifecylce methods)...